lib/vsc/filesystem/posix.py

```diff
@@ -383,7 +383,7 @@ def populate_home_dir(self, user_id, group_id, home_dir, ssh_public_keys):
         @type ssh_public_keys: list of strings representing the public ssh keys
         """
         # ssh
-        self.log.info("Populating home %s for user %s:%s" % (home_dir, user_id, group_id))
+        self.log.info("Populating home %s for user %s:%s", home_dir, user_id, group_id)
```

Comment:
what does it fix?

Reply:
Now that I look at it, that is a good question. The error was not enough values for interpolation.

Reply:
But in any case, this the better format :)